pkg/virt-handler/vm.go

```diff
@@ -639,7 +639,7 @@ func (d *VirtualMachineController) migrationSourceUpdateVMIStatus(origVMI *v1.Vi
 		log.Log.Object(vmi).Infof("migration completed to node %s", migrationHost)
 	}
 
-	if !reflect.DeepEqual(oldStatus, vmi.Status) {
+	if !equality.Semantic.DeepEqual(oldStatus, vmi.Status) {
```

Comment:
fyi, I'm aware that https://github.com/kubevirt/kubevirt/pull/7131 fixes this. I'll rebase once that pr goes in